t/29-backend-generalhw.t

```diff
@@ -133,7 +146,7 @@ subtest 'error handling' => sub {
     $connect_fail = 1;
     $backend->reset_console({testapi_console => 'sut'});
     my $ret = $backend->select_console({testapi_console => 'sut'});
-    is_deeply $ret, {error => "connect_remote failed at t/29-backend-generalhw.t line 66.\n"}, 'Correct error returned';
+    is_deeply $ret, {error => "connect_remote failed at t/29-backend-generalhw.t line 67.\n"}, 'Correct error returned';
```

Comment:
Maybe `like $ret->{error}, …` would make more sense to avoid having to include the line number here. (Of course you're not introducing the code so you can also just leave it like this.)